pkg/core/state/notification_event.go

```diff
@@ -20,6 +20,79 @@ type NotificationEvent struct {
 	Item       *stackitem.Array `json:"state"`
 }
 
+// ContractInvocation contains method call information.
+// The Arguments field will be nil if serialization of the arguments exceeds a predefined limit
+// (for security reasons). In that case Truncated will be set to true.
```

Comment:
It's a documentation, we need presize number for the predefined limit to be specified. Use a link to the constant.